app/main.py

```diff
@@ -1,3 +1,69 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+
+from app.car import Car
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open(
+            "app/config.json",
+            "rb"
+    ) as datafile:
+        data = json.load(datafile)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers = data["customers"]
+    shops = data["shops"]
+
+    for customer in customers:
+        cus = Customer(**customer)
```

Comment:
Be more descriptive in variable naming, there is no need to shorten variable names.